xbmc/utils/URIUtils.h

```diff
@@ -67,8 +82,8 @@ class URIUtils
          strExtensions is empty.
    \sa GetExtension
    */
-  static bool HasExtension(const std::string& strFileName, const std::string& strExtensions);
-  static bool HasExtension(const CURL& url, const std::string& strExtensions);
+  static bool HasExtension(const std::string& strFileName, std::string_view strExtensions);
+  static bool HasExtension(const CURL& url, std::string_view strExtensions);
```

Comment:
Could you please remove the type prefix from the string params?

=> https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#87-methods